### PR TITLE
modules index: added ngx_mruby

### PR DESCRIPTION
**Pull request by selmertsx**
I added ngx_mruby module to modules/index.rst.
ngx_mruby is embedded mRuby script language for nginx-module.

### CODE REVIEW

File: source/modules/index.rst [hunk 1]
```diff
@@ -138,6 +138,8 @@ NGINX 3\ :sup:`rd` Party Modules
 +---------------------------+--------------------------------------------------------------------------+-----------------------------------------------------------------------------------------+
 | Mongo                     | Upstream module for direct communication with MongoDB                    | :github:`simpl/ngx_mongo`                                                               |
 +---------------------------+--------------------------------------------------------------------------+-----------------------------------------------------------------------------------------+
+| mRuby                     | Embedded mRuby script language for nginx-module                           | :github:`matsumotory/ngx_mruby`                                                        |
```
Comment (kou) [inline source/modules/index.rst]: It should be `mruby` not `mRuby`.

Comment (selmertsx) [inline source/modules/index.rst]: thank you!!

**Comment by SirNexus** [comment]
Looks good @selmertsx! However, the module is slightly out of alphabetical order. I believe it should be after the mp4_streaming module, not before it. If you would like, I can move it.

**Comment by selmertsx** [comment]
Thank you for reviewing to my PR :grin: SirNexus

> If you would like, I can move it.

Since I did not notice this comment, I fixed my PR 😅

**Comment by SirNexus** [comment]
Perfect, looks great! Thank you!

**Pull request merged**